app/src/main/java/com/example/habitsmasher/ItemAdapter.java

```diff
@@ -29,4 +29,9 @@ public int getItemCount() {
         return _snapshots.size();
     }
 
+    @Override
+    public void updateOptions(@NonNull FirestoreRecyclerOptions<A> options) {
+        super.updateOptions(options);
+        _snapshots = options.getSnapshots();
+    }
```

Comment:
This is needed so that the request is actually removed when a user accepts/denies it